roles/hostedbm/tasks/validations.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Ensure required variables are defined
+  ansible.builtin.assert:
+    that:
+      - "{{ item }} is defined"
+    fail_msg: "The variable {{ item }} is not defined. Please define it to proceed."
+  loop:
+    - hostedbm_cluster_name
+    - hostedbm_cluster_base_domain
+    - hostedbm_guest_ingress_ip
+    - hostedbm_kubeconfig_file
+
+- name: Make sure working directory doesn't exits from previous deployment
```

Comment:
You can collapse this task as well by using:
```
state: "{{ item }}"
loop:
  - absent
  - directory
```

Reply:
done